src/components/tests/Dapp.test.jsx

```diff
@@ -66,13 +66,13 @@ test('updates the petname when pressing enter on the textfield', () => {
 
   let textField = component.find(TextField);
   act(() =>
-    textField.props().onChange({ target: { value: 'Token Palace 2' } }),
+    textField.props().onChange({ target: { value: 'Token Palace 2' } })
```

Comment:
the removal of the comma, seems like Prettier is lost. I'm working on fixing CI. meanwhile, please `prettier --write` over the PR

Reply:
fixed